boards/nrf52dk/Makefile.dep

```diff
@@ -2,6 +2,12 @@ ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
 
-ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
-  USEPKG += nordic_softdevice_ble
+ifneq (,$(USEBLE))
```

Comment:
Ups, this was for my testing...

Will change this back to BLE, as I think that's the better default, right?